### ISSUE TITLE
Density correction files not found after PR #445 on certain Linux environments

### ISSUE THREAD

**Issue by ojalaj** [nrc-cnrc/EGSnrc#485]
With the latest `develop` branch with default configuration options, I tested with EX16MVp an interactive run in pegsless mode and got:

**To the screen:**
```
PHOTONUCLEAR REQUESTED FOR MEDIUM NUMBER  4
 
PHOTONUCLEAR REQUESTED FOR MEDIUM NUMBER  5
 
PHOTONUCLEAR REQUESTED FOR MEDIUM NUMBER  6
 
  PEGSLESS INPUT.  CALCULATING ELECTRON CROSS-SECTIONS.
  density_file                                                                                                                                                                                                                               
 
***************** Error:
Error: Cannot open density correction file:
.density
 
***************** Quiting now.
 ```
 
**To the .mederr file under egsrun directory:**
 ```
  For medium: AIR521ICRU
  Composition specified in material data file
  Rho specified in material data file
  GASP specified in material data file
  Density correction file specified in material data file.
  Error: Density correction file.density                                                                                                                                                                                                    $
  does not exist in
  $EGS_HOME/pegs4/density_corrections,
  $EGS_HOME/pegs4/density_corrections/elements,
  $EGS_HOME/pegs4/density_corrections/compounds,
  $EGS_HOME/pegs4/density,
  $HEN_HOUSE/pegs4/density_corrections/elements or
  $HEN_HOUSE/pegs4/density_corrections/compounds.
 ```

**And to the .errors file under egsrun directory:**
```
*************ERROR*************
VALUE SOUGHT: bremsstrahlung correction
INPUT--><--NOT ALLOWED
OPTIONS ARE:
                                      KM
                                     NRC
                                    NONE
                          OFF IN REGIONS
                                    EADL
                                  SIMPLE
```

The culprit seems to be PR #445 and the 80 --> 256 changes therein. Reverting back to 80, everything runs as expected. Interestingly, on another cluster this issue cannot be reproduced. I have an access to both of the clusters and can provide more details on the systems/environments.

**Comment by blakewalters** [comment]
Hi @ojalaj: Is this still causing you problems?  Could you please send me the input file for the EX16MVp that you're using?  Thanks.

**Comment by ojalaj** [comment]
Well - it's not causing problems, since I reverted back to 80, with which everything seems work as expected. I'll send you the input file in a minute.

**Comment by ftessier** [comment]
@ojalaj , I am trying this on commit d0b1ec9, the tip of the `develop` branch (i.e., including PR #455 from commit 6222e77). It seems to work as expected. Here is what I did:

1. Start from a fresh BEAM_EX16Mvp accelerator and `.egsinp` file (from `/home/ftessier/EGSnrc/HEN_HOUSE/omega/beamnrc/BEAMnrc_examples/EX16MVp`).

2. Copy the `$HEN_HOUSE/pegs4/data/material.dat` file to the accelerator directory `$EGS_HOME/BEAM_EX16Mvp` (otherwise use the full path name to the material data file in step 3 below; both work for me).

3. Add the following code at the end of the `.egsinp` file:
    ```
    :start media definition:

        ae = 0.521
        ap = 0.010
        ue = 50.521
        up = 50

        material data file = material.dat

    :stop media definition:
    ````
4. Run with `BEAM_EX16MVp -i EX16MVp.egsinp`.

If that does not work at your end, then please send me your input file (and material.dat file if modified; check with `git diff $HEN_HOUSE/pegs4/data/material.dat`).

**Comment by rtownson** [comment]
This was fixed in PR #510, which is now merged in so I'm closing this issue.
